src/schema/city/index.ts

```diff
@@ -50,10 +50,20 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           defaultValue: "CURRENT",
           description: "By default we filter shows by current",
         },
+        includeStubShows: {
+          type: GraphQLBoolean,
+          description: "Whether to include local discovery stubs",
+        },
+        displayable: {
+          type: GraphQLBoolean,
+          defaultValue: true,
+          description: "Filter by displayable shows",
```

Comment:
The changes in the last commit are good, but I think I was also suggesting getting rid of this arg, since it's now superfluous, right?

If so let's delete L57-61 ✂️

Reply:
oh i thought i did 🤦‍♂️ let me do it now

Reply:
done! (or i think "done")